plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -66,6 +67,19 @@
   clone:
     description:
       - Name of VM to be cloned. If C(vmid) is setted, C(clone) can take arbitrary value but required for initiating the clone.
+  cicustom:
+    description:
+      - cloud-init: Specify custom files to replace the automatically generated ones at start.
```

Comment:
```suggestion
      - cloud-init: Specify custom files to replace the automatically generated ones at start.
    type: ...
    version_added: 1.0.0
```
Fill in the correct type for `type`. The same changes are needed for all new options.

Reply:
version_added: '1.3.0'